content/es/load_balancer.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Balanceador de Carga
+status: Feedback Appreciated
+category: concept
+---
+
+## Qué es
```

Comment:
There is a new suggestion for template headers in 2c9bc465434c15210917ab995cf0fdb33016ba58

```suggestion
## ¿Qué es?
```